Assets/MixedRealityToolkit/Boundary/MixedRealityBoundarySystem.cs

```diff
@@ -26,6 +28,7 @@ public MixedRealityBoundaryManager()
         public override void Initialize()
         {
             // TODO Initialize stuff 
+            inputSystem = Internal.Managers.MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>();
```

Comment:
We may or may not need this yet in the class. Let's add if only if needed for now.

Reply:
Agreed, this is only there as a "Starter for 10" , I hope to make this a Custom Editor in the future